changelog.md

```diff
@@ -107,6 +106,10 @@
   use the Nim VM in a native Nim application.
 - Added the parameter ``val`` for the ``CritBitTree[T].incl`` proc.
 - The proc ``tgamma`` was renamed to ``gamma``. ``tgamma`` is deprecated.
+- The unicode module's `isUpper` and `isLower` procedures as well as
```

Comment:
* misleading msg since `unicode .isLower(Rune)` is not deprecated
* maybe simpler written as:
```
unicode.[isUpper, isLower](string) is deprecated, etc
```